package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@hapipal/schwifty",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

This gets bumped via the release/tagging process 👍 
```suggestion
  "version": "6.0.0",
```